src/services/category.js

```
@@ -0,0 +1,33 @@
+const Category = require('~/models/category')
+
+const { createError } = require('~/utils/errorsHelper')
+
+const { DOCUMENT_ALREADY_EXISTS } = require('~/consts/errors')
+
+const categoryService = {
+  createCategory: async (data) => {
+    const {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)